data/brands/shop/bag.json

```diff
@@ -53,7 +53,7 @@
     {
       "displayName": "Coach",
       "id": "coach-bdf41b",
-      "locationSet": {"include": ["ca", "us"]},
+      "locationSet": {"include": ["at", "be", "ca", "de", "es", "fr", "gb", "id", "ie", "it", "jp", "mx", "nl", "nz", "us"]},
```

Comment:
The European site gives a list of a number of countries, others have just been random guesswork. I guess we could try every country code or something?

Reply:
Have now merged @Cj-Malone 's ATP data dump into this list, so closing this one for now.